modules/dicom_archive/README.md

```diff
@@ -0,0 +1,43 @@
+# Dicom Archive
+
+## Purpose
+
+The DICOM Archive module provides a front end to view the metadata
+about DICOMs which have been processed by the LORIS database via
+the imaging pipeline. It includes both failed and successful scans.
+
+## Intended Users
+
+The DICOM Archive is primarily used by DCC staff who are investigating
+the status of imaging pipeline scripts.
+
+## Scope
+
+The DICOM Archive shows DICOM tar metadata. It does not provide a
+way to visualize scans or view metadata of other data formats. It
+provides a frontend to the `tarchive_*` tables in LORIS, but does
+not directly inspect files on the filesystem.
+
+## Permissions
+
+The permission `dicom_archive_view_allsites` is required to access
+the DICOM Archive module.
+
+## Configurations
+
+The `patientNameRegex`, `LegoPhantomRegex`, and `LivingPhantomRegex`
+configuration variables provide regular expressions to ensure that
+the DICOM has been properly anonymized. If at least one of these
+isn't matched, the Patient Name is displayed as "INVALID - HIDDEN".
+
```

Comment:
I don't know if that's the intended behaviour, or just because a blank regex matches everything.

Reply:
I only mention this because for projects that do not touch this field in the configuration module, some default behavior happens (and I have seen in some demos for projects that has PatientID information which displayed some identifying info because they only anonymized the PatientName field); so it might be a good place to let projects know that they have the option  to replace it by Hidden by doing the proper setting.

Reply:
ok, good point.

Reply:
I added something.